lib/bolt/task.rb

```diff
@@ -9,6 +9,9 @@ def initialize(target, task)
   end
 
   class Task
+    STDIN_METHODS       = %w[both stdin].freeze
```

Comment:
Is this going to eventually end up in the shell base class?

Reply:
It might make sense for it to eventually be a predicate method (maybe `accepts_stdin?`) on `Task` instead of constants that need to be interpreted by the transport.

I was sort of leaving it for a bit to see how the other transports/shells shake out.